codebases/index.md

```diff
@@ -0,0 +1,37 @@
+
+# Where we work: our codebases
```

Comment:
I think the word _where_ is confusing in this sentence, it keeps confusing me and makes me think of physical locations. For example I keep thinking, 'Dutch municipalities'.

```suggestion
# What we work on: our codebases
```

Reply:
I've removed the 5 Ws as being too contrived, after getting that feedback from multiple people.